docs/usage/model_config.md

```diff
@@ -610,7 +612,9 @@ try:
 except NameError as e:
     print(e)
     """
-    Field "also_protect_field" has conflict with protected namespace "also_protect_"
+    Field "also_protect_field" has conflict with protected namespace "also_protect_".
+
+    You may be able to resolve this error by setting `model_config['protected_namespaces'] = ('protect_me_',)`.
```

Comment:
We could also only show this extra bit of the error message when `model_config['protected_namespaces'] == ('model_',)` but I don't think it hurts anything to show the full info, I could see it being useful if the protected namespaces were set on some parent class